AccountData/build.gradle

```diff
@@ -23,28 +23,31 @@ android {
 }
 
 dependencies {
-  testCompile libs.junit
-  testCompile libs.assertjCore
-  testCompile libs.mockitoCore
-  testCompile libs.robolectric
-  testCompile "com.squareup.okhttp3:mockwebserver:$okHttpVersion"
-
-  compile libs.supportAnnotations
-  compile libs.rxjava
-  compile libs.retrofitConverterGson
-  compile libs.retrofitAdapterRxJava
-  compile libs.retrofit
-  compile libs.gson
-  provided project(":AccountDomain")
-  provided project(":TripKitDomain")
-
-  compile libs.dagger
+  testImplementation libs.junit
+  testImplementation libs.assertjCore
+  testImplementation libs.mockitoCore
+  testImplementation libs.robolectric
+  testImplementation "com.squareup.okhttp3:mockwebserver:$okHttpVersion"
```

Comment:
Let's add `"com.squareup.okhttp3:mockwebserver:$okHttpVersion"` into the dependencies file, yay?

Reply:
It is already there :)

Reply:
OK then let's use it here instead, yay?

Reply:
yes